Game/Source/GDKTestGymsFunctionalTests/GDKTestGymsFunctionalTests.cpp

```diff
@@ -2,8 +2,20 @@
 
 #include "GDKTestGymsFunctionalTests.h"
 #include "LogGymsSpatialFunctionalTest.h"
-#include "Modules/ModuleManager.h"
+#include "TestMapGeneration.h"
 
 DEFINE_LOG_CATEGORY(LogGymsSpatialFunctionalTest);
 
-IMPLEMENT_PRIMARY_GAME_MODULE( FDefaultGameModuleImpl, GDKTestGymsFunctionalTests, "GDKTestGymsFunctionalTests" );
+IMPLEMENT_GAME_MODULE(FGDKTestGymsFunctionalTestsModule, GDKTestGymsFunctionalTests)
+
+#define LOCTEXT_NAMESPACE "FGDKTestGymsFunctionalTestsModule"
+
+void FGDKTestGymsFunctionalTestsModule::StartupModule()
+{
+   FCoreDelegates::OnFEngineLoopInitComplete.AddLambda([]() {
+	   SpatialGDK::TestMapGeneration::GenerateTestMaps();
```

Comment:
How does this look (I assume no UI at this point)? How fast is this?

Reply:
It barely seemed to add to the load time

Reply:
Very nice, it probably uses the same speed-up it does for the commandlet.
If you were super-keen, you could look at timestamped logs, or add a timer to the map generation, but just if you want to.